upstream-configuration/composer.json

```diff
@@ -80,7 +80,8 @@
         "drupal/block_field": "1.0.0-rc2",
         "asuwebplatforms/webspark-module-webspark_cas": "1.0.1",
         "asuwebplatforms/webspark-module-webspark_webdir": "1.0.20",
-        "drupal/field_states_ui": "2.0"
+        "drupal/field_states_ui": "2.0",
+        "drupal/ctools": "3.7.0"
     },
     "version": "0.1.104"
```

Comment:
This looks good, but we will need to increment the version number. After you approve mine, it will be "0.1.106" I think.